model_zoo/vision/alexnet.py

```diff
@@ -2,16 +2,12 @@
 import argparse
 from os.path import join
 import google.protobuf.text_format as txtf
-import lbann
-from lbann.util import lbann_dir
-import lbann.models
-import lbann.proto
-import lbann.contrib.args
+import lbann, lbann.models, lbann.proto, lbann.contrib.args
```

Comment:
Minor thing, but [PEP8 recommends imports be on separate lines](https://www.python.org/dev/peps/pep-0008/#imports).